polars/polars-core/src/chunked_array/strings.rs

```diff
@@ -21,6 +51,13 @@ impl Utf8Chunked {
         ca.rename(self.name());
         Ok(ca)
     }
+    /// Extract json path, first match
+    /// Refer to https://goessner.net/articles/JsonPath/
+    #[cfg(feature = "extract_jsonpath")]
+    pub fn extract_json_path_single(&self, json_path: &str) -> Result<Utf8Chunked> {
+        let pat = Compiled::compile(json_path).unwrap();
```

Comment:
Let's match the error and return a `ValueError` if compilation fails. Than we don't panic.

Reply:
I should throw some error now.